### ISSUE TITLE
unable to select nested positions (multiple positions under the same company)

### ISSUE THREAD

**Issue by alexcaulfield** [linkedtales/scrapedin#56]
Hi - thank you for putting this together, I'm excited to use it!

The issue I am having at the moment is that when I am scraping my personal profile and since I have multiple positions at the same company, it's not giving a clean data structure for that company. What is returning is one value with the company name and the most recent description, and then there are 2 separate values for the individual positions, that don't have the company name, and the Title has "Title" preceding the name of the position.

My profile is https://www.linkedin.com/in/alexandercaulfield/

Response
```
[ { title: 'Wayfair',
[0]     location: 'Boston, MA',
[0]     description:
[0]      '- Mentor for 25 entry level engineers on a month long project, holding office hour sessions and giving code reviews (for PHP & React)',
[0]     date1: 'Jul 2019 – Present',
[0]     date2: '5 mos',
[0]     roles: [ [Object], [Object] ] },
[0]   { title: 'Title\nSoftware Engineer II',
[0]     location: 'Boston, MA',
[0]     description:
[0]      '- Mentor for 25 entry level engineers on a month long project, holding office hour sessions and giving code reviews (for PHP & React)',
[0]     date1: 'Jul 2019 – Present',
[0]     date2: '5 mos',
[0]     roles: [ [Object] ] },
[0]   { title: 'Title\nSoftware Engineer I',
[0]     location: 'Boston, MA',
[0]     description:
[0]      '- Full stack development for Wayfair\'s customer-facing storefront (Shop the Look) in React, PHP, SQL and GraphQL\n- Developed the backend for a large A/B test that resulted in reduced exit rate on the page by 3% and an improved bounce rate by -4.5% by leveraging recommended content to users based on their likes and dislikes\n- Tech lead the implementation of a feature on the new page redesign. This included scoping out tickets, gathering requirements with product, constructing the frontend architecture, executing on creating the UI and delegating tasks to other engineers when appropriate\n- Proactive about sharing new ideas for Shop the Look\'s product and supported ideas with data and customer insights\n...\nsee more',
[0]     date1: 'Sep 2018 – Jul 2019',
[0]     date2: '11 mos',
[0]     roles: [ [Object] ] },
```
Let me know if I can do anything to help!

**Comment by romanzubenko** [comment]
@alexcaulfield Can confirm, it returns pretty messy results for any nested roles

**Comment by leonardiwagner** [comment]
Fixed that on `v1.0.11` , please update your scrapedin version.

**NOTE:**  If a position has `roles` you have to ignore its `description`  because it'll be the same of 1st role item.

Your results on 1.0.11:
```javascript
"positions": [
    {
      "title": "Wayfair",
      "location": "Boston, MA",
      "description": "- Mentor for 25 entry level engineers on a month long project, holding office hour sessions and giving code reviews (for PHP & React)",
      "date1": "Jul 2019 – Present",
      "date2": "6 mos",
      "roles": [
        {
          "title": "Software Engineer II",
          "description": "- Mentor for 25 entry level engineers on a month long project, holding office hour sessions and giving code reviews (for PHP & React)",
          "date1": "Jul 2019 – Present",
          "date2": "6 mos",
          "location": "Boston, MA"
...
```

Please give feedback

**Comment by leonardiwagner** [comment]
Closing due to lack of feedback
